### ISSUE TITLE
Standard variables do not exist

### ISSUE THREAD

**Issue by Sidstumple** [MatiseAms/henris#2]
## Problem
`$font-medium` f.e. does not exist.

## Proposal 
Since it's not possible to declare variables, maybe use `font-weight: weight(medium);`

```
$font-weights: (
	"bold": 800,
	"medium": 600,
	"regular": 400,
	"light": 200
);

@each $weight, $weight-value in $font-weights {
	.font-#{$weight} {
		font-weight: $weight-value;
	}
}

```

**Comment by silvandiepen** [comment]
I think we should implement something like this into the typography as an option to create automatically.

Thinking about an option to include either in the big scss variable include (ex. in `$font-primary`) like
```
$font-primary: (
   "font-weight-classes": true
);
```
Or a separate `$output-font-weights: true`. When there  a no font-weights set, this will not output, also not when $output-css: false;

**Comment by silvandiepen** [comment]
Font weight implementation with classes and function is included from 0.3.5
